### PR TITLE
[FrameworkBundle] Implemented ChainCacheClearer service

### PR DESCRIPTION
**Pull request by giovannialbero1992**
integration for (https://github.com/jymfony/jymfony/pull/11)

### CODE REVIEW

File: src/Component/Kernel/src/CacheClearer/CacheClearerInterface.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/**
+ * @memberOf Jymfony.Component.Kernel.CacheClearer
+ */
+class CacheClearerInterface {
+    /**
+     * Clears any caches necessary
+     *
+     * @param {String} cacheDir
```
Comment (massimilianobraglia) [inline src/Component/Kernel/src/CacheClearer/CacheClearerInterface.js]: jsdoc type string has the initial 's' in lowercase

File: src/Component/Kernel/src/CacheClearer/ChainCacheClearer.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+const CacheClearerInterface = Jymfony.Component.Kernel.CacheClearer.CacheClearerInterface;
+
+/**
+ * @memberOf Jymfony.Component.Kernel.CacheClearer
+ */
+class ChainCacheClearer extends implementationOf(CacheClearerInterface) {
+    /**
+     * @param {[Jymfony.Component.Kernel.CacheClearer.CacheClearerInterface} clearers
```
Comment (massimilianobraglia) [inline src/Component/Kernel/src/CacheClearer/ChainCacheClearer.js]: remove wrongly opened '['

File: src/Component/Kernel/src/CacheClearer/ChainCacheClearer.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+     * @inheritDoc
+     */
+    clear(cacheDir) {
+        for(let clearer of this._clearers) {
+            clearer.clear(cacheDir);
+        }
+    }
+
+    /**
+     *
```
Comment (massimilianobraglia) [inline src/Component/Kernel/src/CacheClearer/ChainCacheClearer.js]: remove empty comment line

**Review by massimilianobraglia — CHANGES_REQUESTED** [review]

File: src/Component/Kernel/src/CacheClearer/ChainCacheClearer.js (hunk 2 above)
Comment (giovannialbero1992) [reply src/Component/Kernel/src/CacheClearer/ChainCacheClearer.js]: In this case is missed a ']' because the argument is an array.

Comment (massimilianobraglia) [reply src/Component/Kernel/src/CacheClearer/ChainCacheClearer.js]: 👍

**Comment by massimilianobraglia** [comment]
@giovannialbero1992 FINALLY, it is merged 😄

**Comment by giovannialbero1992** [comment]
Great @massimilianobraglia

**Pull request merged**